<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterWindow</class>
 <widget class="QDialog" name="FilterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="unfilterCourseLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Unfiltered Courses</string>
   </property>
  </widget>
  <widget class="QPushButton" name="allButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>92</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ALL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noneButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>92</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>NONE</string>
   </property>
  </widget>
  <widget class="QLabel" name="filterCourseLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtered Courses</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>92</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>------&gt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="unfilteredList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>231</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="filteredList">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>231</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>92</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;------</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>420</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
